next/components/forms/segments/AccountActivator/AccountActivator.tsx

```diff
@@ -8,7 +8,7 @@ const AccountActivator = () => {
   const { t } = useTranslation('account')
   return (
     <AccountContainer className="mb-0">
-      <div className="flex flex-col gap-2 rounded-xl bg-gray-50 px-5 py-4 md:rounded-none md:bg-gray-0 md:p-0">
+      <div className="md:bg-gray-0 flex flex-col gap-2 rounded-xl bg-gray-50 px-5 py-4 md:rounded-none md:p-0">
```

Comment:
Seems like wrong ordering - breakpoint values are usually at the end, unknown classes are usually at the start.

Reply:
Thanks for pointing out the sort issues, I realized I've provided wrong path to Tailwind theme CSS.
03c39214112cdde179b8312b2f096392cc2f71c3